aspect/intellij_info_impl.bzl

```diff
@@ -216,10 +216,12 @@ def collect_py_info(target, ctx, semantics, ide_info, ide_info_file, output_grou
 
 def _collect_generated_proto_go_sources(target):
     """Returns a depset of proto go source files generated by this target."""
-    if not hasattr(target, "aspect_proto_go_api_info"):
-        return None
-    go_proto_info = target.aspect_proto_go_api_info
-    files = getattr(go_proto_info, "files_to_build", [])
+    files = []
```

Comment:
This is not necessary, variables can be declared in the if scope, and still be used outside of the scope.